.github/workflows/flutter_news_example.yaml

```diff
@@ -28,7 +28,7 @@ jobs:
 
     strategy:
       matrix:
-        channel: [stable, beta]
```

Comment:
A new `beta` is about to drop, i'd suggest not making this change.

Reply:
Was just verifying the other jobs are passing. I don’t intend to merge this change 👍